.github/workflows/soundness.yml

```diff
@@ -47,6 +47,10 @@ on:
         type: boolean
         description: "Boolean to enable the shell check job. Defaults to true."
         default: true
+      shell_check_container_image:
+        type: string
+        description: "Container image for the shell check job. Defaults to latest Swift Ubuntu image."
+        default: "swift:5.10-noble"
```

Comment:
Do we really want to make this configurable?

Reply:
Just wanted to put it up here so that when these get bumped, this one doesn't get forgotten, which it might if it was declared inline.